### PR TITLE
{bio}[foss/2020b] GADMA v2.0.0rc12 w/ Python 3.8.6

### PR DESCRIPTION
**Pull request by jfgrimm**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by branfosj** [comment]
Test report by @branfosj
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux RHEL 8.3, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/058de95869781c1d6604745ebb84346f for a full test report.

File: easybuild/easyconfigs/g/GADMA/GADMA-2.0.0rc12-foss-2020b.eb [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    ('ruamel.yaml.clib', '0.2.2', {
+        'modulename': False,
+        'checksums': ['2d24bd98af676f4990c4d715bcdc2a60b19c56a3fb3a763164d2d8ca0e806ba7'],
+    }),
+    ('ruamel.yaml', '0.17.4', {
+        'checksums': ['44bc6b54fddd45e4bc0619059196679f9e8b79c027f4131bb072e6a22f4d5e28'],
+    }),
+    ('nlopt', '2.6.2', {
+        'checksums': ['21f10b0bbcf32b4d370c371e967c01df50ce96c258f483a32b9140b57e9e410d'],
+    }),
```
Comment (branfosj) [inline easybuild/easyconfigs/g/GADMA/GADMA-2.0.0rc12-foss-2020b.eb:42]: The failure that I am seeing is during the building of `nlopt`: https://gist.github.com/branfosj/a59ab25d0813dac9fa6455eb9574bcba

The error is unclear, but I think there is a missing dependency.

**Review by branfosj — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/g/GADMA/GADMA-2.0.0rc12-foss-2020b.eb (hunk 1 above)
Comment (jfgrimm) [reply easybuild/easyconfigs/g/GADMA/GADMA-2.0.0rc12-foss-2020b.eb:42]: Can you try building the NLopt-python easyconfig and see if that works?

**Comment by boegel** [comment]
closing this since `foss/2020b` is deprecated, see https://docs.easybuild.io/policies/toolchains

**Pull request closed without merging**